CHANGELOG.rst

```diff
@@ -4,6 +4,138 @@ SeqAn Changelog
 This file summarizes the changes to the SeqAn library and apps.
 
 
+Release 2.1.0
+~~~~~~~~~~~~~
+
+Major release with many new features and applications. 
+Note, the majority of the modules are backward compatible to the previous version.
```

Comment:
Replace with: "Except where noted below, this release is compatible to previous 2.x releases."